copier/main.py

```diff
@@ -513,6 +517,36 @@ def _render_file(self, src_abspath: Path) -> None:
             dst_abspath.write_bytes(new_content)
             dst_abspath.chmod(src_mode)
 
+    def _render_symlink(self, src_abspath: Path) -> None:
+        assert src_abspath.is_absolute()
+        src_relpath = src_abspath.relative_to(self.template_copy_root)
```

Comment:
I think I commented it previously... Absolute links should be kept absolute IMHO.

Reply:
This doesn't change the target of the symlink. This is the path where the symlink itself is.
The handling of the symlink target is in the code block below